Sources/NIO/Channel.swift

```diff
@@ -337,6 +337,23 @@ public enum ChannelLifecycleError: Error {
     case inappropriateOperationForState
 }
 
+/// This should be inside of `ChannelError` but we keep it separate to not break API.
+// TODO: For 2.0: bring this inside of `ChannelError`.
```

Comment:
mind filing an issue for that with the 2.0 milestone? Just so we won't forget and then we get nice github support of how many outstanding things

Reply:
Sure, see #620.